mlir/lib/Dialect/Vector/IR/VectorOps.cpp

```diff
@@ -5575,6 +5575,34 @@ LogicalResult ShapeCastOp::verify() {
   return success();
 }
 
+namespace {
```

Comment:
Probably a case for `static`: https://llvm.org/docs/CodingStandards.html#restrict-visibility

Reply:
Thanks. I intend to change this in a follow-up PR.